x-pack/filebeat/module/activemq/log/_meta/fields.yml

```diff
@@ -0,0 +1,15 @@
+- name: log
+  type: group
+  description: >
+    Fields from ActiveMQ application logs.
+  fields:
+    - name: caller
+      type: keyword
+      description: >
+        Name of the caller issuing the logging request (class or resource).
+    - name: thread
+      type: keyword
+      description: >
+        Thread that issued the logging request.
+    - name: java_exception
```

Comment:
This exception could go to the ECS field [`error.stack_trace`](https://www.elastic.co/guide/en/ecs/1.2/ecs-error.html).

Reply:
I'm not sure if it's easily doable. The testing framework verifies the processed event with successful expected events. Any error that occurs, so do `error.stack_trace`, fails whole testing scenario.

The idea is great, I believe we just lacks `_test_expected_errors` method.

Reply:
We should do something about this sooner than later, we are not using some ECS fields that make a lot of sense as common fields because of a limitation in the test framework. This remembers me this issue: https://github.com/elastic/beats/issues/13052

Would you like to give a try to solve this? :innocent:

Reply:
Let's postpone the decision of using a common field till we have a solution for #13052.